assets/src/containers/FilesAccessed.js

```diff
@@ -14,21 +23,120 @@ const styles = theme => ({
   paper: {
     padding: theme.spacing.unit * 2,
     color: theme.palette.text.secondary
+  },
+  formController: {
+    display: "flex",
+    marginTop: theme.spacing.unit * 2,
+    alignItems: "center",
+    justifyContent: "center"
   }
 })
 
 function FilesAccessed (props) {
   const { classes, match } = props
-  const currentCourseId = match.params.courseId
-  const [loaded, assignmentData] = useFilesAccessedAssignmentData(currentCourseId)
+  const currentCourseId = match.params.courseId;
+  const [minMaxWeek, setMinMaxWeek] = useState([1, 17]);
```

Comment:
this information should be coming from the `/info` endpoint call. look for `total_weeks` attribute in the json response
`{"canvas_id": "233333", "term_id": 1770000000000034, "name": "CO 999 002 FA 2018", "term": {"id": 17700000000000134, "canvas_id": "111", "name": "Fall 2018", "date_start": "2018-09-04 00:00:00", "date_end": "2018-12-31 00:00:00"}, "current_week_number": 33, "total_weeks": 16, "course_view_options": {"fa": 1, "ap": 1, "gd": 0}}`

Reply:
done